packages/site/index.html

```diff
@@ -10,7 +10,7 @@
   <title>Atlantis</title>
   <link rel="icon" type="image/svg+xml" href="/atlantis_favicon.svg" />
   <link rel="preconnect" href="https://cdn.jobber.com">
-  <link rel="prefetch" href="https://cdn.jobber.com/fonts/fonts.css" as="style">
+  <link rel="stylesheet" href="https://cdn.jobber.com/fonts/fonts.css">
```

Comment:
I can understand why the prefetch change was made. I did some light reading and seems like `preload` could also be used for fonts but that didn't work when I tried it.
The direction in our internal docs is to have this `<link>` as you currently have it with the `stylesheet`. So if it's good enough for everyone, it's good enough for me too.